setup.cfg

```diff
@@ -1,5 +1,75 @@
 [metadata]
+name = pytest-xdist
+description = pytest xdist plugin for distributed testing and loop-on-failing modes
+long_description = file: README.rst
+license = MIT
+author = holger krekel and contributors
+author_email = pytest-dev@python.org,holger@merlinux.eu
+url = https://github.com/pytest-dev/pytest-xdist
+platforms =
+    linux
+    osx
+    win32
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Framework :: Pytest
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: POSIX
+    Operating System :: Microsoft :: Windows
+    Operating System :: MacOS :: MacOS X
+    Topic :: Software Development :: Testing
+    Topic :: Software Development :: Quality Assurance
+    Topic :: Utilities
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
 license_file = LICENSE
 
+[options]
+packages = find:
+package_dir = =src
+zip_safe = False
+python_requires = >=3.6
+install_requires =
+    execnet>=1.1
+    pytest>=6.2.0
+    pytest-forked
+setup_requires = setuptools_scm>=6.0
+
+[options.packages.find]
+where = src
+
+[options.entry_points]
+pytest11 =
+    xdist = xdist.plugin
+    xdist.looponfail = xdist.looponfail
+
+[options.extras_require]
+testing =
+    filelock
+    pytest>=6.2.0
+psutil = psutil>=3.0
+setproctitle = setproctitle
+
 [flake8]
 max-line-length = 100
+
+[mypy]
+mypy_path = src
+# check_untyped_defs = True
```

Comment:
I think we should remove the commented options, or leave a comment explaining why they should be left uncommented.